features/DeviceController.feature

```diff
@@ -352,3 +352,10 @@ Feature: Device Manager device controller
       | collection | "payloads"       |
     Then I should receive a result matching:
       | total | 1 |
+
+  Scenario: Import devices using csv
+    Given I successfully execute the action "device-manager/device":"importDevices" with args:
```

Comment:
I don't understand how importing the string `qos.battery,measures.temperature.degree\\n80,23.3` can create the device `DummyTemp-attached_ayse_unlinked` (which already exists and so an error should be thrown instead here)